src/org/labkey/mobileappstudy/MobileAppStudyManager.java

```diff
@@ -1238,7 +1275,36 @@ private void deleteParticipantDataFromSurveyLists(Participant participant) throw
         Collection<ListDefinition> lists = ListService.get().getLists(participant.getContainer()).values();
 
         for (ListDefinition list :lists)
-            deleteParticipantFromList(list, participant.getContainer(), participant.getRowId(), user);
+        {
+            Container container = participant.getContainer();
+
+            if (list.getName().equalsIgnoreCase(PARTICIPANT_PROPERTIES_LIST_NAME))
+                deleteParticipantPropertiesFromList(list, container, getEnrollmentToken(container, participant.getRowId()), user);
+            else
+                deleteParticipantFromList(list, container, participant.getRowId(), user);
+        }
+    }
+
+    /**
+     * Delete participant data from ParticipantProperties list using EnrollmentToken
+     * @param list participantProperties ListDefinition
+     * @param container hosting study
+     * @param enrollmentToken token associated to participant
+     * @param user authorized for delete permissions
+     * @throws Exception
+     */
+    private void deleteParticipantPropertiesFromList(ListDefinition list, Container container, String enrollmentToken, User user) throws Exception
+    {
+        //Get the table
+        TableInfo table = list.getTable(user, container);
+        if (table == null)
+            throw new NotFoundException("Unable to find table for list '" + list.getName() + "' in container '" + container.getName() + "'");
```

Comment:
This was chosen for consistency since we use the NotFoundException throughout the MobileAppStudyManager for similar search actions: e.g., getResultTable. I think it would be appropriate to switch it broadly.

Reply:
Changed with https://github.com/LabKey/mobileAppStudy/pull/82/commits/57c3f0a83d5bcdc2f591f20bb16ac8e9b4b73790